### PR TITLE
[1LP][RFR]V2v schedule migration tests

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Adding tests__ for schedule_migration feature.

**Note**: Please ignore the changes in Widgetastic_manageiq/__init__.py as these are already being reviewed in #7818 
Once that PR is merged, this PR should be reviewed and merged.

{{pytest: cfme/tests/v2v/test_v2v_schedule_migrations.py --use-provider rhv42 --use-provider vsphere6-nested --provider-limit 2 -vvvv --long-running}}

### CODE REVIEW

**Comment by kedark3** [comment]
To be merged after #7866

File: cfme/tests/v2v/test_v2v_schedule_migrations.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_VERSION,
+        fixture_name='second_provider',
+        required_flags=['v2v']
+    )
+]
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
```
[the remark this answers is not in the document]
Comment (kedark3) [reply cfme/tests/v2v/test_v2v_schedule_migrations.py]: If you look at the `form_data_vm_obj_single_datastore` fixture, it needs source Datastore Type and Target Datastore type. In this case both are `nfs`, but both could be different in other tests, maybe `nfs` and `iscsi` combinations.

[the remark this answers is not in the document]
Comment (sshveta) [reply cfme/tests/v2v/test_v2v_schedule_migrations.py]: :+1:

**Comment by sshveta** [comment]
Waiting for PRT results

**Comment by kedark3** [comment]
PRT failure seems to be due to something wrong with schedule feature in 5.9.5 build Checking further.

File: cfme/tests/v2v/test_v2v_schedule_migrations.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from cfme.fixtures.provider import rhel7_minimal
```
Comment (mshriver) [inline cfme/tests/v2v/test_v2v_schedule_migrations.py]: No need to import this, the module is registered with pytest in `cfme/test_framework/pytest_plugin`

Comment (kedark3) [reply cfme/tests/v2v/test_v2v_schedule_migrations.py]: I believed I had this comment by some one on my v2v PR, and I tried removing this, but didn't work.

**Comment by kedark3** [comment]
@mshriver please review this as possible.

**Pull request merged**